minio/xml_marshal.py

```diff
@@ -66,3 +66,134 @@ def xml_marshal_complete_multipart_upload(uploaded_parts):
         s3_xml.ElementTree(root).write(data, encoding=None,
                                        xml_declaration=False)
     return data.getvalue()
+
+def xml_marshal_bucket_notifications(notifications):
+    """
+    Marshals the notifications structure for sending to S3 compatible storage
+
+    :param notifications: Dictionary with following structure:
+
+    {
+        'TopicConfigurations': [
+            {
```

Comment:
Need to decide on an appropriate place to put the in-code documentation for the structure expected by the set_bucket_notification() API and returned by get_bucket_notification() API.

The expected structure is indicated here at this point.